src/main/java/com/gdsc/pikpet/entity/Shelter.java

```diff
@@ -0,0 +1,20 @@
+package com.gdsc.pikpet.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+public class Shelter {
+    @Column(nullable = false)
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int shelterId;
```

Comment:
```suggestion
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(nullable = false, name = "shelter_id")
    private int id;
```
헷갈릴 거 같으니까 필드명 id로 통일하는 게 좋을 거 같은데 어떻게 생각하시나요~
그리구 PK는 shelter_id 형식으로 테이블_id 로 DB에 넣는 게 좋을 거 같은데 어떻게 생각하시나요~

Reply:
그게 저도 좋은 것 같아요~